newIDE/app/src/UI/EmptyPlaceholder.js

```diff
@@ -48,22 +52,36 @@ export const EmptyPlaceholder = (props: Props) => (
         </Text>
         <LargeSpacer />
         <ColumnStackLayout alignItems="center" noMargin>
-          <RaisedButton
-            label={props.actionLabel}
-            primary
-            onClick={props.onAction}
-            disabled={!!props.isLoading}
-            icon={
-              props.isLoading ? (
-                <CircularProgress size={24} />
-              ) : props.actionIcon ? (
-                props.actionIcon
-              ) : (
-                <Add />
-              )
-            }
-            id={props.actionButtonId}
-          />
+          <Line noMargin>
```

Comment:
I don't know if this should be a ResponsiveLineStackLayout. Could you add a story for the component with a secondary action?

Reply:
The 2nd button only appear according to the clipboard content. I don't know a good way to force it. I guess it can be shown by copying a behavior on the story that have ones first.

Reply:
I meant adding a story for the component EmptyPlaceholder, it should be straightforward